### PR TITLE
Hotfix/Consumer Default Options

### PR DESCRIPTION
**Pull request by Antwan86**
This PR removes duplicate default configuration for consumer options.
Now it is considering command line arguments only if they are present, other wise it just default to the defaults as set in `consumer_options.py` dictionary.

Addresses #187 #202 #214

### CODE REVIEW

**Comment by camilonova** [comment]
Pretty good. Thank you.

File: huey/contrib/djhuey/management/commands/run_huey.py [hunk 1]
```diff
@@ -67,8 +65,6 @@ def add_arguments(self, parser):
                     short = '-V'
                 if 'type' in kwargs:
                     kwargs['type'] = self._type_map[kwargs['type']]
-                if kwargs.get('default') is not True:
-                    kwargs.pop('default', None)
```
Comment (coleifer) [inline huey/contrib/djhuey/management/commands/run_huey.py]: I would've thought this (and the lines above it) would have been sufficient to clear out the defaults?

Comment (Antwan86) [inline huey/contrib/djhuey/management/commands/run_huey.py]: Unless the defaults are true (this is why `utc` and `no_periodic` are not used).
It's not worth having the defaults set in the optparser in the first place at all, which is why I suggested to remove them.

**Comment by camilonova** [comment]
@Antwan86 Please fix conflicts.

**Comment by coleifer** [comment]
Let's do it!

**Comment by coleifer** [comment]
Thanks

**Comment by camilonova** [comment]
Thank you

**Pull request merged**